datanode/gateway/graphql/schema.graphql

```diff
@@ -1943,6 +1943,14 @@ type Market {
 
   "Optional: Market ID of the successor to this market if one exists"
   successorMarketID: ID
+
+  "Funding Periods for perpetual markets"
```

Comment:
```suggestion
  "Funding periods for perpetual markets"
```

Reply:
I agree - this would be good to have in the field description. Based on https://github.com/vegaprotocol/vega/blob/97980ebf5ba285076bf3a5eb5a4e0c23706df08f/datanode/gateway/graphql/schema.graphql#L287-L288 I guess it's a RFC3339Nano?